ports/esp32/usb.h

```diff
@@ -26,6 +26,7 @@
 #ifndef MICROPY_INCLUDED_ESP32_USB_H
 #define MICROPY_INCLUDED_ESP32_USB_H
 
+#define USB_CDC_TIMEOUT  (500)
```

Comment:
Please rename this to `MICROPY_HW_USB_CDC_TX_TIMEOUT`, so it's consistent with other config variables named a similar way.

Reply:
Done. The initial commits are updated.